### PR TITLE
Fixed Error while Kill Tor

### PR DESCRIPTION
**Pull request by muratulashozturk**
Changed some pieces of stuff for fix. I checked the code. It killing the process successfully. Please check it.

### CODE REVIEW

File: Tor_KillTor_code.js [hunk 1]
```diff
@@ -14,8 +14,8 @@ PROCESS_STANDART_OUTPUT = _result()
 sleep(1000)!
 native("filesystem", "removefile", RANDOM_FILE)
 PID = PROCESS_STANDART_OUTPUT.split('pid=')[1].split(' ')[0]
-RANDOM_FILE = "temp_" + rand() + ".bat"
```
Comment (katant) [inline Tor_KillTor_code.js:17]: ?

File: Tor_KillTor_code.js [hunk 2]
```diff
@@ -26,5 +26,5 @@ PROCESS_ERROR_OUTPUT = base64_decode(split[1])
 {
 PROCESS_STANDART_OUTPUT = _result()
 }
-sleep(1000)!
```
Comment (katant) [inline Tor_KillTor_code.js:29]: ?

**Review by katant — COMMENTED** [review]
Explain those changes

File: Tor_KillTor_code.js (hunk 1 above)
Comment (muratulashozturk) [reply Tor_KillTor_code.js:17]: It's bugging when creating it again. So adding a prefix for the filename for killing is solving the issue.

File: Tor_KillTor_code.js (hunk 2 above)
Comment (muratulashozturk) [reply Tor_KillTor_code.js:29]: It's for waiting more time. I am thinking that the remove file is executing early.

File: Tor_KillTor_code.js (hunk 1 above)
Comment (muratulashozturk) [reply Tor_KillTor_code.js:17]: Maybe It has a problem with the BAS. Have some problems with the **rand()** function. Maybe not working well for the twice calling. That's why I added the **Kill** prefix.

Comment (muratulashozturk) [reply Tor_KillTor_code.js:17]: Also adding **cmd /U** is making bat file cmd with UNICODE options. So It can successfully kill the task.

**Comment by katant** [comment]
I will check it later

**Comment by katant** [comment]
The error is in `PID = PROCESS_STANDART_OUTPUT.split('pid=')[1].split(' ')[0]`, because Netcat can't get Tor pid. But all your changes are below 16. They can't help at all

**Comment by muratulashozturk** [comment]
Actually, you don't need Netcat @katant You can use this cmd code to get PID: 
**netstat -ao | findstr [[TOR_PORT]]**
**Output is: `TCP    127.0.0.1:9507         0.0.0.0:0              LISTENING       12208`**
you should only get the first line because sometimes It can be more than 1 in the output, 
Then you can use Regex to get the only PID.
```
let netstat = "TCP    127.0.0.1:9507         0.0.0.0:0              LISTENING       12208";
let pidNum = netstat.match('(\\d+)$');
let torPid = pidNum[0];
```

**Comment by katant** [comment]
I already found initial issue. Also netstat your doesn't exit after execution. I will release fix today/tomorrow

**Pull request closed without merging**